_schemaV2.graphql

```diff
@@ -13288,6 +13289,12 @@ type Query {
     # The ID of the Credit Card
     id: String!
   ): CreditCard
+  curatedMarketingCollections(
+    after: String
+    before: String
+    first: Int
+    last: Int
+  ): [MarketingCollection]
```

Comment:
**suggestion**: Since we're returning an array of MarketingCollection entries here, it seems disingenuous to use the after/before/first/last connection-style args. I think we should expose `size: Int` instead and pass that through to the delegated query.

Reply:
makes sense, I think I wanted to still follow connection conventions, but it doesn't makes sense I guess